public_wiki/LocalSettings.php

```diff
@@ -148,7 +148,17 @@
 wfLoadSkin( 'Vector' );
 wfLoadSkin( 'MonoBook' );
 wfLoadSkin( 'MinervaNeue' );
-$wgDefaultSkin = 'minerva';
+#$wgDefaultSkin = 'minerva';
+# Set default theme depending on browser
+# Adding Mobile Detect extension
+wfLoadExtension( 'MobileDetect' );
+$mobile = wfMobileDetect();
+if ( $mobile ) {
+    $wgDefaultSkin = "chick"; # If mobile
+} else {
+    $wgDefaultSkin = "vector"; # If not mobile
+}
+
```

Comment:
```suggestion
$wgDefaultSkin = 'vector';

# Adding Mobile Detect extension
wfLoadExtension( 'MobileDetect' );

# Update default theme depending on browser
if (wfMobileDetect()) {
    $wgDefaultSkin = 'minerva'; # switch if mobile
}
```

Reply:
I downloaded the version for 1.35, MobileDetect-REL1_35-c137344.tar

Happy for the code suggestion. I'd taken the if/else code from the wikimedia page.